vscode-html/compile-vscode-html-languageserver.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+# @see https://github.com/mattn/vim-lsp-settings/pull/48
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="${SCRIPT_DIR}"
+SRC_DIR="${REPO_DIR}/vscode-html-languageserver"
+DIST_DIR="${REPO_DIR}/out"
+
+
+# ------------------------- #
+# download the source codes #
+# ------------------------- #
+
+pushd "${REPO_DIR}" || exit
+
+rm -rf "${SRC_DIR}" "${DIST_DIR}" "package.json" "package-lock.json"
+
+# or get the source via git clone
+# git clone --depth=1 https://github.com/vscode-langservers/vscode-html-languageserver "${SRC_DIR}"
+
+wget https://github.com/vscode-langservers/vscode-html-languageserver/archive/master.zip -O src.zip
+unzip src.zip
+rm -f src.zip
+mv vscode-html-languageserver-master "${SRC_DIR}"
+
+popd || exit
+
+
+# ------------ #
+# prepare deps #
+# ------------ #
+
+pushd "${SRC_DIR}" || exit
+
+npm install
+npm install -D typescript
+cat << EOF > tsconfig.json
+{
+    "compilerOptions": {
+        "target": "es2018",
+        "module": "commonjs",
+        "strict": true,
+        "alwaysStrict": true,
+        "noImplicitAny": true,
+        "noImplicitReturns": true,
+        "noUnusedLocals": true,
+        "noUnusedParameters": true,
+        "outDir": "./out"
+    },
+    "files": [
+        "src/htmlServerMain.ts"
+    ]
+}
+EOF
+
+popd || exit
+
+
+# ------- #
+# compile #
+# ------- #
+
+pushd "${SRC_DIR}" || exit
+
+./node_modules/typescript/bin/tsc --newLine LF -p .
+
+cp package.json "${REPO_DIR}"
+cp package-lock.json "${REPO_DIR}"
```

Comment:
This will copy `package-lock.json` created when building so with all dev dependencies included. I think we need one without them.
We need one that we can create by running `npm i --production` in `vscode-html` directory.

Reply:
hmm... in my experiment, `npm i --production` won't remove devDeps' version info from `package-lock.json`

Reply:
Hmm, that's true.
So it's the `dev: true` property in lock file that is used for marking dev dependencies.
So feel free to do it the previous way - it should be a bit faster.
Sorry again for creating confusion. :)